framework/packages/abstract-integration-tests/src/account_factory.rs

```diff
@@ -21,7 +21,7 @@ pub fn create_one_account_with_namespace_fee<T: MutCwEnv>(mut chain: T) -> AResu
     chain
         .set_balance(&sender, vec![namespace_fee.clone()])
         .unwrap();
-    version_control.update_config(None, None, Some(Clearable::Set(namespace_fee.clone())))?;
+    version_control.update_config(None, Some(Clearable::Set(namespace_fee.clone())), None)?;
```

Comment:
It will shoot everywhere, should we disable alphabet sorting of fields? It will be pretty breaking change for everything in next version, but more stable later

Reply:
I think it's ok for it to shoot everywhere so we pay a little more attention to the changes